### DISCUSSION TITLE
Step file issues

### DISCUSSION THREAD

**Discussion by anthonybenelisha** [stevenbennett/DIYson#15] [Q&A] [answered]
Hello,

Sorry for the probably very basic issue, hopefully this is the right place to ask it. I cannot figure out how to download the step files for the diyson express. I see the diyson has its own stl files. I can see the raw step text, but I cannot download a stp file directly. I can only copy the text and cannot figure out what do with it after that.

Thank you in advance for the help.

**Comment by clemensul** [comment]
Hi - you can download the file as `.txt` and just rename it to `.step` on Windows.

If you are using a MAC you have to open the text file in TextEdit and use saveAs to change it to a step-File.
(source: https://forums.autodesk.com/t5/fusion-360-design-validate/converting-txt-to-step-tried-renaming/m-p/9658385#M223802)

Hope that helps. :)

**Comment by momonunu** [comment]
Maybe 3mf files would also be a good idea

**Comment by stevenbennett** [comment, marked as answer]
@anthonybenelisha I know it's a bit of a pain but I would suggest just downloading the whole project, that will get you all of the STEP files in their native format.
<img width="952" alt="Screenshot 2023-02-03 at 10 27 38 AM" src="https://user-images.githubusercontent.com/4053041/216655768-6fff5afd-22ac-4ce0-abfb-a7dad8d86fcc.png">

**Reply by anthonybenelisha** [reply]
Yes that works. Sorry for my confusion, new to github. Thank you for the quick reply. The project is awesome. 

Thanks,
Anthony

**Reply by stevenbennett** [reply]
All good, it's really not the most straightforward process honestly. Have fun with those files!!
